Dangerfile

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+github.dismiss_out_of_range_messages
+
+# `files: []` forces rubocop to scan all files, not just the ones modified in the PR
+# `skip_bundle_exec` prevents RuboCop from running using `bundle exec`, which we don't want on the linter agent
+rubocop.lint(files: [], force_exclusion: true, inline_comment: true, fail_on_inline_comment: true, include_cop_names: true, skip_bundle_exec: true)
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/Gravatar-SDK-Android/blob/2805492494021bbdb70339073544abc0e00048f2/Dangerfile#L7"></span>Layout/LineLength: Line is too long. [147/120]</td>
    </tr>
  </tbody>
</table>